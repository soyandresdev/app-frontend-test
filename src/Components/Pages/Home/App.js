import React from "react";
import SideMenu from "../../Molecules/SideMenu";
import { Container } from "./stylesHome";
function App() {
  return (
    <Container>
      <SideMenu />
      <div>
        <h1>Aqui el formulario</h1>
      </div>
    </Container>
  );
}

export default App;
