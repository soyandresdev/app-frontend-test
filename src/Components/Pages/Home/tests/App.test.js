import React from "react";
import renderer from "react-test-renderer";
import App from "../App";
// import { Container } from "../stylesHome";

describe("Test component [App]", () => {
  it("Renders correctly defaultProps [App]", () => {
    const rendered = renderer.create(<App />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
