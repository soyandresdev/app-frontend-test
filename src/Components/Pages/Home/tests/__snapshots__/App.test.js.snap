// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component [App] Renders correctly defaultProps [App] 1`] = `
<div
  className="sc-htoDjs juvQWM"
>
  <nav
    className="sc-bdVaJa dzzEKa"
  >
    <div>
      <figure
        className="sc-bwzfXH jaTVMM"
      >
        <img
          alt="Logo Addi"
          src="logo_addi.png"
        />
      </figure>
      <ul
        className="sc-htpNat ffIfPS"
      >
        <li
          className="sc-bxivhb fcqCpQ"
        >
          <a
            className="sc-ifAKCX gfFhMs"
            href="#"
          >
            <svg
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <rect
                fill="none"
                height="24"
                rx="0"
                ry="0"
                width="24"
              />
              <path
                clip-rule="evenodd"
                d="M15.8 2C15.3582 2 15 2.35817 15 2.8V5.1L12.2867 3.20071C12.1146 3.0802 11.8854 3.0802 11.7133 3.20071L2.21327 9.85071C2.07961 9.94428 2 10.0972 2 10.2603V11.0397C2 11.4442 2.45534 11.6813 2.78673 11.4493L5 9.9V21.2C5 21.6418 5.35817 22 5.8 22H18.2C18.6418 22 19 21.6418 19 21.2V9.9L21.2133 11.4493C21.5447 11.6813 22 11.4442 22 11.0397V10.2603C22 10.0972 21.9204 9.94428 21.7867 9.85071L19 7.9V2.8C19 2.35817 18.6418 2 18.2 2H15.8ZM17.8 7.06V3.6C17.8 3.37909 17.6209 3.2 17.4 3.2H16.6C16.3791 3.2 16.2 3.37909 16.2 3.6V5.94L17.8 7.06ZM17.8 9.06V20.4C17.8 20.6209 17.6209 20.8 17.4 20.8H6.6C6.37909 20.8 6.2 20.6209 6.2 20.4V9.06L11.7133 5.20071C11.8854 5.0802 12.1146 5.0802 12.2867 5.20071L17.8 9.06Z"
                fill="#ffffff"
                fill-rule="evenodd"
              />
            </svg>
            <span
              className="sc-EHOje eSNhqD"
            >
              Home
            </span>
          </a>
        </li>
        <li
          className="sc-bxivhb fcqCpQ"
        >
          <a
            className="sc-ifAKCX gfFhMs"
            href="#"
          >
            <svg
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <rect
                fill="none"
                height="24"
                rx="0"
                ry="0"
                width="24"
              />
              <path
                clip-rule="evenodd"
                d="M15.0005 12.1302C16.3528 11.1808 17.2432 9.61566 17.2432 7.84131C17.2432 4.94971 14.8916 2.59814 12 2.59814C9.1084 2.59814 6.75684 4.94971 6.75684 7.84131C6.75684 9.61566 7.64722 11.1808 8.99951 12.1302C5.55298 13.0823 2.40039 16.7125 2.40039 20.1792C2.40039 20.854 2.94824 21.4019 3.62305 21.4019H20.377C21.0518 21.4019 21.5996 20.854 21.5996 20.1792C21.5996 16.7125 18.447 13.0823 15.0005 12.1302ZM7.95605 7.84131C7.95605 5.61182 9.77051 3.79736 12 3.79736C14.2295 3.79736 16.0439 5.61182 16.0439 7.84131C16.0439 10.0618 14.2449 11.8673 12.0289 11.8833H11.9711C9.75513 11.8673 7.95605 10.0618 7.95605 7.84131ZM20.377 20.2026L3.59961 20.1792C3.59961 16.7964 7.31348 13.0825 10.6963 13.0825H11.9709C11.981 13.0826 11.99 13.0854 12 13.0854C12.01 13.0854 12.019 13.0826 12.0291 13.0825H13.3037C16.6865 13.0825 20.4004 16.7964 20.4004 20.1792C20.4004 20.1919 20.3896 20.2026 20.377 20.2026Z
"
                fill="#ffffff"
                fill-rule="evenodd"
              />
            </svg>
            <span
              className="sc-EHOje eSNhqD"
            >
              Clientes
            </span>
          </a>
        </li>
        <li
          className="sc-bxivhb fcqCpQ"
        >
          <a
            className="sc-ifAKCX gfFhMs"
            href="#"
          >
            <svg
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <rect
                fill="none"
                height="24"
                rx="0"
                ry="0"
                width="24"
              />
              <path
                clip-rule="evenodd"
                d="M17.9 3C17.6239 3 17.4 3.22386 17.4 3.5V5.4H15.5C15.2239 5.4 15 5.62386 15 5.9V6.1C15 6.37614 15.2239 6.6 15.5 6.6H17.4V8.5C17.4 8.77614 17.6239 9 17.9 9H18.1C18.3761 9 18.6 8.77614 18.6 8.5V6.6H20.5C20.7761 6.6 21 6.37614 21 6.1V5.9C21 5.62386 20.7761 5.4 20.5 5.4H18.6V3.5C18.6 3.22386 18.3761 3 18.1 3H17.9ZM3.38673 17.2929C3.73862 14.8296 5.84822 13 8.33648 13H11.6635C14.1518 13 16.2614 14.8296 16.6133 17.2929L17 20C17 20 15 21 10 21C5 21 3 20 3 20L3.38673 17.2929ZM4.52841 17.6005C4.78296 15.5773 6.5187 14.0695 8.55761 14.1003L11.6214 14.1466C13.6585 14.1774 15.347 15.7345 15.5425 17.7624L15.6843 19.234C15.6843 19.234 14 19.8 10 19.8C6 19.8 4.32289 19.234 4.32289 19.234L4.52841 17.6005ZM14 8C14 10.2091 12.2091 12 10 12C7.79086 12 6 10.2091 6 8C6 5.79086 7.79086 4 10 4C12.2091 4 14 5.79086 14 8ZM12.8 8C12.8 9.5464 11.5464 10.8 10 10.8C8.4536 10.8 7.2 9.5464 7.2 8C7.2 6.4536 8.4536 5.2 10 5.2C11.5464 5.2 12.8 6.4536 12.8 8Z"
                fill="#ffffff"
                fill-rule="evenodd"
              />
            </svg>
            <span
              className="sc-EHOje eSNhqD"
            >
              Aliados
            </span>
          </a>
        </li>
        <li
          className="sc-bxivhb fcqCpQ"
        >
          <a
            className="sc-ifAKCX gfFhMs"
            href="#"
          >
            <svg
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <rect
                fill="none"
                height="24"
                rx="0"
                ry="0"
                width="24"
              />
              <path
                clip-rule="evenodd"
                d="M3 3.2C3 3.08954 3.08954 3 3.2 3H4C4.11046 3 4.2 3.08954 4.2 3.2V19.6C4.2 19.7105 4.28954 19.8 4.4 19.8H20.8C20.9105 19.8 21 19.8895 21 20V20.8C21 20.9105 20.9105 21 20.8 21H3.5C3.22386 21 3 20.7761 3 20.5V3.2ZM18 7.2C18 7.08954 18.0895 7 18.2 7H19.8C19.9105 7 20 7.08954 20 7.2V17.8C20 17.9105 19.9105 18 19.8 18H18.2C18.0895 18 18 17.9105 18 17.8V7.2ZM14.2 3C14.0895 3 14 3.08954 14 3.2V17.8C14 17.9105 14.0895 18 14.2 18H15.8C15.9105 18 16 17.9105 16 17.8V3.2C16 3.08954 15.9105 3 15.8 3H14.2ZM6 8.2C6 8.08954 6.08954 8 6.2 8H7.8C7.91046 8 8 8.08954 8 8.2V17.8C8 17.9105 7.91046 18 7.8 18H6.2C6.08954 18 6 17.9105 6 17.8V8.2ZM10.2 11C10.0895 11 10 11.0895 10 11.2V17.8C10 17.9105 10.0895 18 10.2 18H11.8C11.9105 18 12 17.9105 12 17.8V11.2C12 11.0895 11.9105 11 11.8 11H10.2Z"
                fill="#ffffff"
                fill-rule="evenodd"
              />
            </svg>
            <span
              className="sc-EHOje eSNhqD"
            >
              Reportes
            </span>
          </a>
        </li>
      </ul>
    </div>
    <div>
      <a
        className="sc-bZQynM cQFpzY"
        href="#"
      >
        <a
          className="sc-gzVnrw ilYsO"
        >
          <span>
            A
          </span>
        </a>
        <p>
          Andres Hernandez
        </p>
      </a>
    </div>
  </nav>
  <div>
    <h1>
      Aqui el formulario
    </h1>
  </div>
</div>
`;
